### PR TITLE
Add initial project files

### PR DESCRIPTION
**Pull request by ygoumaz**
ditto.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM cyberbotics/webots:R2022b-ubuntu20.04
```
Comment (BenjaminDeleze) [inline Dockerfile]:
```suggestion
FROM cyberbotics/webots.cloud:R2022b
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,34 @@
...
+RUN mkdir -p $PROJECT_PATH
+COPY . $PROJECT_PATH
+```
+
+The following two additional lines allow to compile the controller and the plugin when [webots.cloud](webots.cloud) creates the Docker image.
+
+```dockerfile
+RUN cd $PROJECT_PATH/controllers/e-puck && make clean && make
+RUN cd $PROJECT_PATH/plugins/remote_controls && make clean && make
+```
```
Comment (BenjaminDeleze) [inline README.md]: I think you also need to compile the robot window.

I don't know if it make a lot of sense to put the remote_controls plugin as the user will not be able to test it in most of the case

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Seems good, some suggestions:

**Review by ThomasOliverKimble — CHANGES_REQUESTED** [review]
Same comments as Benjamin. Minor detail: I would suggest to open the world in Webots and save it again to create a thumbnail. On save it should be a hidden file that would be uploaded to github

**Comment by ygoumaz** [comment]
I have cleaned the repository and added your suggestions. I have tried to start the world on webots.cloud but unfortunately I still get the same error message telling me that it can't find the .wbt file. Can it be related to its name containing a hyphen symbol ('-') ?

**Comment by BenjaminDeleze** [comment]
It could be (even so it should not), could you try to rename the world to test?

**Review by BenjaminDeleze — COMMENTED** [review]
Seems good to me, but I think we should understand and fix the problem before merging

**Comment by ygoumaz** [comment]
Sure we should fix the problem, changing the name to e_puck.wbt still triggers the error message.

**Comment by ThomasOliverKimble** [comment]
>  Can it be related to its name containing a hyphen symbol ('-') ?

This used to be a problem but it was fixed in `simulation.php` here #[3](https://github.com/cyberbotics/webots-cloud/pull/3)

**Comment by ygoumaz** [comment]
@ThomasOliverKimble Could you check why it fails? The only reason I see is a wrong parsing of the URL, as the same project works in your own repo: https://github.com/ThomasOliverKimble/GuidedTour/blob/guided-tour/e-puck/worlds/e-puck.wbt

**Comment by ygoumaz** [comment]
The issue with the world loading has been fixed in [#4874](https://github.com/cyberbotics/webots/pull/4874). The repository is now completely cleaned and works fine on webots.cloud.

**Review by BenjaminDeleze — DISMISSED** [review]
Thank you!

**Comment by ThomasOliverKimble** [comment]
> Sure we should fix the problem, changing the name to e_puck.wbt still triggers the error message.

fixed with webots PR #[4874](https://github.com/cyberbotics/webots/pull/4874)

**Comment by ygoumaz** [comment]
I have replaced the C controller by a Python one. It should be equivalent.

**Review by BenjaminDeleze — APPROVED** [review]
Thank you!

**Review by ThomasOliverKimble — APPROVED** [review]
Thank you!

**Pull request merged**